cmd/tools/rpcwrappers/main.go

```diff
@@ -275,14 +289,27 @@ func makeGetMatchingClient(reqType reflect.Type) string {
 		// Always route these requests to the same matching node for all namespaces.
 		tq = fieldWithPath{path: "\"not-applicable\""}
 		tqt = fieldWithPath{path: "enumspb.TASK_QUEUE_TYPE_UNSPECIFIED"}
-		nsID = fieldWithPath{path: `"not-applicable"`}
+		nsID = fieldWithPath{path: "\"not-applicable\""}
```

Comment:
change the others to backticks instead? it seems nicer than the backslashes

Reply:
Sure